### ISSUE TITLE
will their be pi 5 support?

### ISSUE THREAD

**Issue by syfer1987** [toolboc/psx-pi-smbshare#87]
Hi, can I ask to stream games to ps3 via muli man but the issue is pi 3 is hard to find and pi 4 8gb is easy to find but pi 5 same price Pi 4. So I want to know if possible to use pi 5 8gb before pi 4 gets discontinued.  Also, their no info any were if 8 GB RAM is good for streaming games.  Need advise.

**Comment by AlphaUserJeff0001** [comment]
Hi @syfer1987 i have my pi5 setup and i have exactly 8gb ram as you so it works ( btw. i use Bookworm Newest Edition 64x ) :)

**Comment by AlphaUserJeff0001** [comment]
Yes it's good sharing those games but i recommend better Ethernet cable ( CAT6 or better. but CAT8 is the best like overkill )  so 8gb is very good
